samples/ingestion/ingestion-client/infra/main.json

```diff
@@ -4,11 +4,26 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.28.1.47646",
-      "templateHash": "7855477000736253469"
+      "version": "0.29.47.4906",
+      "templateHash": "10746629345153305500"
     }
   },
   "parameters": {
+    "EnvironmentName": {
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 20,
+      "metadata": {
+        "description": "Name of the the environment which is used to generate a short unique hash used in all resources."
+      }
+    },
+    "ResourceToken": {
```

Comment:
what's the advantage of using ResourceToken here instead of DeploymentId with default utcNow()?

I think it's more likely that customers would accidentally create resources with the same ResourceToken if they pick an EnvironmentName themselves.. Maybe we should rename it keep it as DeploymentId with default utcNow(), and then generate the hash based on `[toLower(uniqueString(subscription().id, parameters('DeploymentId'), parameters('AzureSpeechServicesRegion')))]` if that makes sense to you?

Reply:
OK - this was actually an upcoming change that should probably be done in a separate PR so I have pulled this out so we can concentrate on Code Coverage which this PR is about. I have left the shortened Function Names in as that is a warning in App Service at the moment (but also happy to pull that out into a separate PR too)